packages/api/test/e2e/fees.e2e.ts

```diff
@@ -50,12 +50,8 @@ describe('fees in cennznet', () => {
               if (status.isFinalized && events !== undefined) {
                 const blockHash = status.asFinalized;
                 const events = ((await api.query.system.events.at(blockHash)) as unknown) as Vec<EventRecord>;
-                let gas;
-                for (const {event} of events) {
-                  if (event.method === 'Charged') {
-                    gas = event.data[1];
-                  }
-                }
+                const feeChargeEvent = events.find(event => event.event.data.method === 'Charged');
+                const gas = feeChargeEvent.event.data[1];
```

Comment:
`gas` is not the right term, can change to `realFee` or similar (I know you didn't write this originally but since we're changing it already)

The expect should read more like:
` expect(realFee.toString()).toEqual(calculatedFee.toString());`
same goes for the other usages